test/context.go

```diff
@@ -48,6 +48,7 @@ func GetEnvContext() *Context {
 	c.TPPZone = os.Getenv("TPP_ZONE")
 	c.TPPZoneRestricted = os.Getenv("TPP_ZONE_RESTRICTED")
 	c.TPPZoneECDSA = os.Getenv("TPP_ZONE_ECDSA")
+	c.TPPaccessToken = os.Getenv("TPP_ACCESS_TOKEN")
```

Comment:
Why are you adding this? We are already setting an Access Token at the beginning of our test, basing on username and password, here: https://github.com/Venafi/vcert/blob/master/pkg/venafi/tpp/connector_test.go#L78

I'd suggest you remove it (and do necessary code changes in tests so you don't use it), as adding this new variable will impact on our pipeline

Reply:
removed